src/commons.ts

```diff
@@ -131,7 +131,17 @@ export default class Commons {
         const tempObj = JSON.parse(customSettingStr);
 
         Object.assign(customSettings, tempObj);
-        customSettings.token = customSettings.token.trim();
+        if (customSettings.token) {

```

Comment:
This looks good. Lets keep this. But Will it really help in terms of current exception users are facing ?